### DISCUSSION TITLE
最想去的迪士尼樂園

### DISCUSSION THREAD

**Discussion by THH1118** [1101-GS4719-C/Python-Community-GS4719-C#1678] [General]
全球現在的迪士尼樂園有6間，分別是香港、上海、東京、美國加州、美國奧蘭多以及法國。自己本身有去過其中兩個(東京跟奧蘭多)，藉著需要發文的機會來簡單盤點這兩個樂園哪個更好。

(1)	園區設施:奧蘭多>>東京
東京的迪士尼是分為迪士尼樂園以及迪士尼海洋
奧蘭多則是有四個園區:神奇王國、EPCOT、好萊屋影城還有動物王國
我覺得奧蘭多大勝的原因是選擇真的非常多，而且其實裡面有很多大人取向很刺激的設施，東京的設施就比較for更年輕的學生(但還是不錯啦只是個人更喜歡玩很刺激的那種)。

(2)	價位:奧蘭多(貴)>東京(便宜)
以單一園區來算
東京的門票約是2000台幣左右
奧蘭多的則是約3000左右
但如果是快速通關的話是東京的比較貴

(3)	其他因素
距離:東京>>奧蘭多
排隊程度:東京>奧蘭多(真的不要小看日本人的排隊能力www)
氣候:東京>奧蘭多(奧蘭多的天氣其實跟台灣很像，夏天幾乎每天都有一點午後雷陣雨，東京的天氣相對穩定一點)

大概就是這樣，如果我只能選一個去的話，還是會選奧蘭多，雖然做十幾個小時的飛機真的是很累。

**Comment by sagitariolotus** [comment]
我去過上海、香港、東京的迪士尼，個人覺得東京的最好玩，香港迪士尼是這三個中最小的，半天就玩完全部了，東京還分為陸上跟海洋兩個主題，玩了兩天還覺得不夠，上海迪士尼因為選在非熱門時段，完全不需要排隊，遊玩品質不錯，但設施的精彩程度還是偏好東京的！

**Comment by chia-kai-chang** [comment]
我以前有去Orlando參加研討會，但是沒有去迪士尼樂園。
那時還是用小米手機2S

<img width="389" alt="2016-06-09_21-54-01" src="https://github.com/NCU-GS4719-Python/Python-Community/assets/1768891/63e01be2-1ea5-4294-8f17-398b24246eb5">

<img width="389" alt="2016-06-09_21-31-51" src="https://github.com/NCU-GS4719-Python/Python-Community/assets/1768891/6f9369b4-ed63-4ea9-9083-7bcfaa7693c2">
